localstack/services/infra.py

```diff
@@ -352,7 +352,7 @@ def terminate_all_processes_in_docker():
             except Exception:
                 pass
     # kill the process itself
-    sys.exit(0)
```

Comment:
`sys.exit(0)` only exit current thread so that process itself is not exit.

Reply:
it only terminates the interpreter if `sys.exit` is called from the main thread